.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,92 @@
+<!--
+  Make sure you have read CONTRIBUTING.md completely before you file a new
+  issue! 
+
+  If possible, try to determine if the bug is actually part of the Swift driver,
+  or if the issue is actually from `libmongoc` or `libbson`. If so, you should
+  file the issue with the representative projects.
+-->
+
+## Information
+
+### Swift version
+
+What does this command give you?
+```
+$ swift --version
+```
+
+
+### Operating system
+
+What does this command give you?
+```
+$ uname -a
+```
+
+
+### Driver version
+
+What does this command give you?
+```
+$ cat Package.swift # Only applies if you are using Swift package manager
```

Comment:
looking at the `Package.swift` you added for the sample project made me realize, we should actually have people do `cat Package.resolved`. SPM rounds up patches to the latest patch version when installing (so 0.03 -> 0.0.7), as evidenced by the `Package.resolved` you committed.  

but SPM won't automatically install an update when a new one comes out, so without knowing whether they have recently done `swift package update` or if they very recently added the driver as a dependency, we can't tell much from what's stated here -- they could still actually be on 0.0.3 or they could be all the way up to 0.0.7, or anything in between. 

I'm not sure if there is a `Package.resolved` equivalent for Cocoapods, but if so switch to that as well